OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/Optimove.java

```diff
@@ -211,6 +222,28 @@ public static void setCredentials(@Nullable String optimoveCredentials, @Nullabl
         }
     }
 
+    /**
+     * Late setting of credentials. Must be called only if partial initialisation constructor was used and only once.
+     *
+     * @param optimoveCredentials   credentials for track and trigger
+     * @param optimobileCredentials credentials for other mobile features (push, in-app, deep links etc)
+     * @param preferenceCenterCredentials credentials for preference center feature
+     */
+    public static void setCredentials(@Nullable String optimoveCredentials, @Nullable String optimobileCredentials, @Nullable String preferenceCenterCredentials) {
```

Comment:
1. never a fan of copying large chunks of code. Can 2nd overload call 1st overload? + a separate method on OptimoveConfig to set PC creds
2. After we set PC credentials late, PC config is not updated? perhaps simplest is to just store OptimoveConfig in PC to avoid updating many copies